src/frontends/tensorflow_common/src/helper_transforms/embedding_segments_feature_fusing.cpp

```diff
@@ -66,11 +66,10 @@ ov::frontend::tensorflow::pass::EmbeddingSegmentSingleFeatureFusion::EmbeddingSe
     // SparseFillEmptyRows outputs segment ids along with indices for each segment. Indices correspond to vectors from
     // embedding table if some segment ids are not specified, SparseFillEmptyRows generate default indice for this
     // segment
-    auto sparse_fill_empty_rows = make_shared<SparseFillEmptyRows>(gather0_1->output(0),
-                                                                   gather0_2->output(0),
-                                                                   dense_shape_pattern->output(0),
-                                                                   default_value_pattern->output(0));
-
+    auto sparse_fill_empty_rows = make_shared<v16::SparseFillEmptyRows>(gather0_2->output(0),
+                                                                        dense_shape_pattern->output(0),
+                                                                        gather0_1->output(0),
+                                                                        default_value_pattern->output(0));
     auto strided_slice =
         make_shared<v1::StridedSlice>(sparse_fill_empty_rows->output(0),
                                       make_shared<v0::Constant>(element::i32, Shape{2}, vector<int32_t>{0, 0}),
```

Comment:
BTW, can we remove this transformation at all if we support SparseSegmentSum?
I think it is possible to provide translator for SparseSegmentSum. It can be don in separate PR:)

Reply:
I can look into that, but in a separate ticket - this PR is part of 25.2 commitments.